lib/chat/view/ChatScreen.dart

```diff
@@ -48,20 +46,23 @@ class ChatScreen extends StatelessWidget {
         }
 
         _schedulerBinding.addPostFrameCallback((_) {
-          _scrollController.animateTo(
-              _scrollController.position.maxScrollExtent,
-              duration: const Duration(milliseconds: 250),
-              curve: Curves.easeOut);
+          _scrollController.animateTo(_scrollController.position.maxScrollExtent, duration: const Duration(milliseconds: 250), curve: Curves.easeOut);
         });
 
         return Scaffold(
-          appBar: ChatAppBar(
-            title: _role.requestTitle,
-            description: _role.requestDescription,
+          appBar: AppBar(
+            title: Text('Chat'),
```

Comment:
Any reason for swapping the custom appbar? Also description field is missing in the new one.